### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by Demydoff** [abrechts/Phoenix-ELN#17] [General]
![photo_2024-10-02_18-18-01](https://github.com/user-attachments/assets/d0e39884-a821-4e65-bfb2-38211b1ed0a9)
![photo_2024-10-02_18-17-59](https://github.com/user-attachments/assets/8da589f9-78d1-4ed0-a379-4ce4c03a7a8d)
![photo_2024-10-02_18-17-59 (2)](https://github.com/user-attachments/assets/6c789ee9-382a-4ce2-ba01-989b953d74b4)
![photo_2024-10-02_18-18-00](https://github.com/user-attachments/assets/a0dea7a8-905b-421a-9e77-6f77b772b651)

**Comment by abrechts** [comment]
Copied this to Issues section (#18) and changed title to be more descriptive.
